src/main/java/edu/harvard/iq/dataverse/engine/command/impl/ListDataverseTemplatesCommand.java

```diff
@@ -26,14 +26,22 @@ public ListDataverseTemplatesCommand(DataverseRequest request, Dataverse dataver
 
     @Override
     public List<Template> execute(CommandContext ctxt) throws CommandException {
-        List<Template> templates = new ArrayList<>();
+        List<Template> availableTemplates = new ArrayList<>(dataverse.getTemplates());
 
-        if (dataverse.getOwner() != null) {
-            templates.addAll(dataverse.getParentTemplates());
+        if (!dataverse.isTemplateRoot() && dataverse.getOwner() != null) {
+            availableTemplates.addAll(dataverse.getParentTemplates());
```

Comment:
Do we need to make this recursive for multiple generations of owners? While dv.getOwner() != null...